server/bin/pbench-index.py

```diff
@@ -116,6 +81,10 @@ def main(options, name):
         )
         return 2
 
+    _name_suf = "-tool-data" if options.index_tool_data else ""
```

Comment:
This code block should not be moved, the `name` field is referenced in the `print` statement on line 77 above.

Reply:
yeah

Reply:
Looks like this change has not been reverted ...

Reply:
Sorry, forgot to push the changes